common/lib/transport/connectionmanager.js

```diff
@@ -1,9 +1,8 @@
 var ConnectionManager = (function() {
-	var readCookie = (typeof(Cookie) !== 'undefined' && Cookie.read);
-	var createCookie = (typeof(Cookie) !== 'undefined' && Cookie.create);
-	var eraseCookie = (typeof(Cookie) !== 'undefined' && Cookie.erase);
-	var connectionKeyCookie = 'ably-connection-key';
-	var connectionSerialCookie = 'ably-connection-serial';
+	var getFromSession    = (typeof(SessionStorage) !== 'undefined' && SessionStorage.get);
```

Comment:
@mattheworiordan sorry, github hadn't loaded your comment when I posted my last one.

The library uses a pattern of browser-specific helpers in `browser/lib` and node ones in `node/lib` for a bunch of things. I wouldn't be against just having both load all files and using tests & noops, but doing that just for SessionStorage would make that inconsistent with the other browser-specific/node-specific libs, and doing it for everything would be a major refactor at this stage. So I'm inclined to leave it as-is for now